lib/rails-jquery-autocomplete/autocomplete.rb

```diff
@@ -96,12 +96,12 @@ def get_object(model_sym)
     # Can be overriden to show whatever you like
     # Hash also includes a key/value pair for each method in extra_data
     #
-    def json_for_autocomplete(items, method, extra_data=[])
+    def json_for_autocomplete(items, method, extra_data=[], extra_methods=[])
       items = items.collect do |item|
         hash = HashWithIndifferentAccess.new({"id" => item.id.to_s, "label" => item.send(method), "value" => item.send(method)})
-        extra_data.each do |datum|
+        extra_methods.each do |datum|
```

Comment:
you added `extra_methods` (left `extra_data` as it was), but here you have replaced `extra_data` with `extra_methods`. Why?